packages/text-field/character-counter/README.md

```diff
@@ -0,0 +1,54 @@
+# React Text Field Character Counter
+
+MDC React Text Field Character Counter is a React Component which uses [MDC Text Field Character Counter](https://github.com/material-components/material-components-web/tree/master/packages/mdc-textfield/character-counter)'s CSS and foundation JavaScript.
```

Comment:
"...CSS and foundation JavaScript." --> "...Sass and Foundational JavaScript logic."